app/pages/projects-disciplines.cjsx

```diff
@@ -1,15 +1,6 @@
-counterpart = require 'counterpart'
 React = require 'react'
 TitleMixin = require '../lib/title-mixin'
 apiClient = require 'panoptes-client/lib/api-client'
-OwnedCardList = require '../components/owned-card-list-by-discipline'
-
-counterpart.registerTranslations 'en',
```

Comment:
That's true, I have a line in the other PR that removes the OwnedCardList dependency. The removal of the translation stuff is new tho. Will edit so we don't have the same change in 2 PRs.